simulation/main/Model.py

```diff
@@ -83,7 +82,7 @@ def run_simulation_calculations(self) -> None:
         # ----- Tick array -----
 
         self.timestamps = np.arange(0, self.simulation.simulation_duration + self.simulation.tick, self.simulation.tick)
-        self.tick_array = np.diff(self.timestamps)
+        self.tick_array = np.diff(self.timestamps) # can this not just be np.full(self.timestamps.shape, self.simulation.tick)?
```

Comment:
Seems reasonable to me, tick array should just be an array filled with the same value (tick)